mobile/src/components/course/components/edit-course-category/helper/get-default-value.helper.ts

```diff
@@ -0,0 +1,11 @@
+import { DropDownPayload } from '../common/common';
+
+const DEFAUTL_UPDATE_COURSE_CATEGORY_PAYLOAD = null;
+
+const getDefaultValueforDropDown = (
```

Comment:
Remove this helper, use default payload for initialValues, and use reset with useEffect if you need to set value depending on props